tests/test_client.py

```diff
@@ -0,0 +1,178 @@
+from rest_framework import status
+from rest_framework.test import APITestCase
+from django.contrib.auth.models import User
+
+from core import models
+
+
+class ClientTestCase(APITestCase):
+    def setUp(self):
+        User.objects.create_user(
+            username='camilo', email='camilolgon@gmail.com', password='uluyac'
+        )
+
+    def test_create_client(self):
+        client = {
+            "name": "Felicity",
+            "surname": "Jones",
+            "email": "felicity@gmail.com",
+            "telephone": "11984345678",
+            "cpf": "34598712387",
+        }
+        self.client.login(username='camilo', password='uluyac')
```

Comment:
Esse self.client é diferente do dict client aqui de cima? Se for, onde ele é criado/instanciado?

Reply:
The test case inherits from APITestCase, from rest_framework. This client is a APIClient, a member of APITestCase.